flink-ml-benchmark/src/main/java/org/apache/flink/ml/benchmark/Benchmark.java

```diff
@@ -78,6 +79,8 @@ private static void executeBenchmarks(CommandLine commandLine) throws Exception
 
         StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
         env.getConfig().enableObjectReuse();
+        env.getConfig().disableGenericTypes();
```

Comment:
Do we need to `disableGenericTypes` in benchmark? First, not all algorithms work well with this config, e.g. HashingTF. Secondly, it is already guaranteed that kryo is disabled in unit tests of each algorithm.

Reply:
This change can additionally confirm that all DataGenerators have included TypeInformation properly so that the benchmarks can be run with kryo disabled.

Since the purpose of the benchmark is to get the maximum possible performance, I think we will always provide proper type information to make sure that the benchmark can be run with kryo disabled. This includes the benchmark for `HashingTF `.

Reply:
I see, thanks for the explanation.